protostar/commands/migrate/migrate_command.py

```diff
@@ -3,21 +3,22 @@
 from typing import List, Optional
 
 from protostar.cli import Command
-from protostar.commands.deploy import DeployCommand
+from protostar.cli.network_command_mixin import NetworkCommandMixin
+from protostar.cli.signable_command_mixin import SignableCommandMixin
 from protostar.commands.test.test_environment_exceptions import CheatcodeException
-from protostar.migrator import Migrator
+from protostar.migrator import Migrator, MigratorExecutionEnvironment
 from protostar.protostar_exception import ProtostarException
+from protostar.starknet_gateway import GatewayFacade
 from protostar.utils.input_requester import InputRequester
 from protostar.utils.log_color_provider import LogColorProvider
 
 
-class MigrateCommand(Command):
-    GATEWAY_URL_ARG_NAME = "gateway-url"
-    NETWORK_ARG_NAME = "network"
-
+class MigrateCommand(Command, NetworkCommandMixin, SignableCommandMixin):
+    # pylint: disable=too-many-arguments
```

Comment:
i'd remove this check, pretty useless TBH

Reply:
👍🏻 but plz do this in separate PR